src/components/views/rooms/RoomList.tsx

```diff
@@ -71,6 +72,7 @@ interface IProps {
     resizeNotifier: ResizeNotifier;
     isMinimized: boolean;
     activeSpace: SpaceKey;
+    pageType?: PageType;
```

Comment:
it's optional because i am only using it in just 1 special case which is when FavouriteMessagesView is active, also because the typescript check is yelling at me when it was not optional, other files that has nothing to do with PageType that used RoomList Component required me to provide a value. I know i could have put null as value but i still think it's better as optional to avoid altering multiple files.